bc/subscription/models.py

```diff
@@ -125,6 +125,10 @@ class FilingWebhookEvent(AbstractDateTimeModel):
         help_text="The docket id from CL.",
         null=True,
     )
+    cl_doc_id = models.IntegerField(
+        help_text="The document id from CL.",
+        null=True,
+    )
```

Comment:
Calling this `cl_doc_id` is inconsistent vs the docket_id just above. Why isnt' that one cl_docket_id, or why isn't this one just doc_id?

Reply:
You're right, We're not using a namespace for the fields in this model. We should call it `doc_id`.